lib/solarwinds_apm.rb

```diff
@@ -5,69 +5,73 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 begin
+  require 'solarwinds_apm/version'
+  require 'solarwinds_apm/logger'
   if ENV.fetch('SW_APM_ENABLED', 'true') == 'false'
     SolarWindsAPM.logger.warn 'SW_APM_ENABLED environment variable detected and was set to false; SolarWindsAPM disabled'
     return
   end
 
-  require 'solarwinds_apm/version'
-  require 'solarwinds_apm/thread_local'
-  require 'solarwinds_apm/support_report'
-  require 'solarwinds_apm/constants'
-  require 'solarwinds_apm/api'
-  require 'solarwinds_apm/base'
-  require 'solarwinds_apm/logger'
-  require 'solarwinds_apm/config'
-
-  SolarWindsAPM::Config.load_config_file
-  SolarWindsAPM.loaded = false
```

Comment:
It seems safer to keep this initialization of `loaded` to false, any reason to remove?

Reply:
yeah, I changed the structure of SolarWindsAPM module (oboe_metal.rb) that initialize the loaded as [false](https://github.com/solarwinds/apm-ruby/pull/115/files#diff-d6701cc379164d2319e70cbc7f52f3b678d5a1368abb5fdfb4df6c63d5e6c63dR12); so its default will be false as always. It will change to true once reporter initialization process finished without any issue.